composer.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "drupal/redirect",
+  "description": "Sandbox for Drupal 8 Redirect Port",
```

Comment:
I thought it's common to just use `drupal/MODULE` like [Media Entity](https://github.com/drupal-media/media_entity/blob/8.x-1.x/composer.json) did it. Am I wrong here?